package.json

```diff
@@ -1,55 +1,25 @@
 {
   "name": "create-reason-react-app",
-  "private": true,
-  "version": "0.0.0",
-  "description": "",
+  "version": "0.1.0",
+  "description": "reason react project generator",
```

Comment:
Can we name this Create Reason React App?
and the other `package.json` - Reason React App